autolens/plot/plotters/fit_imaging_plotters.py

```diff
@@ -0,0 +1,292 @@
+from autoarray.plot.plotters import abstract_plotters
+from autoarray.plot.plotters import inversion_plotters
+from autoarray.plot.plotters import fit_imaging_plotters
+from autolens.plot.plotters import ray_tracing_plotters
+from autogalaxy.plot.mat_wrap import lensing_mat_plot, lensing_include, lensing_visuals
+from autolens.fit import fit as f
+
+import numpy as np
+
+
+class FitImagingPlotter(fit_imaging_plotters.FitImagingPlotter):
+    def __init__(
+        self,
+        fit: f.FitImaging,
+        mat_plot_2d: lensing_mat_plot.MatPlot2D = lensing_mat_plot.MatPlot2D(),
+        visuals_2d: lensing_visuals.Visuals2D = lensing_visuals.Visuals2D(),
+        include_2d: lensing_include.Include2D = lensing_include.Include2D(),
+    ):
+
+        super().__init__(
+            fit=fit,
+            mat_plot_2d=mat_plot_2d,
+            include_2d=include_2d,

```

Comment:
Given that this constructor only calls super, unless it does something special with default arguments, it may be unnecessary